src/psycop/common/model_evaluation/binary/subgroup_data.py

```diff
@@ -39,11 +39,20 @@ def get_auroc_by_input_df(
     if bin_continuous_input:
         df[f"{input_name}_binned"], _ = bin_continuous_data(df[input_name], bins=bins)
 
-    output_df = auroc_by_group(
-        df=df,
-        groupby_col_name=f"{input_name}_binned",
-        confidence_interval=confidence_interval,
-    )
+        output_df = auroc_by_group(
+            df=df,
+            groupby_col_name=f"{input_name}_binned",
+            confidence_interval=confidence_interval,
+        )
+
+    else:
+        df["n_in_bin"] = df.groupby(input_name)[input_name].transform("count")
+
+        output_df = auroc_by_group(
+            df=df,
+            groupby_col_name=input_name,
+            confidence_interval=confidence_interval,
+        )
```

Comment:
I don't understand this; if we're not binning, why does "n_in_bin" make sense? :-)

Reply:
n_in_group instead? because we are grouping

Reply:
and because i would like the number in groups for plotting (sex)